test/easyconfigs/easyconfigs.py

```diff
@@ -282,12 +282,14 @@ def check_dep_vars(self, dep, dep_vars):
                 # medaka 1.1.* requires TensorFlow 2.2.0 (while other 2019b easyconfigs use TensorFlow 2.1.0 as dep);
```

Comment:
can you also update the comment here to include medaka 1.2?

Reply:
Doen